src/apirequest.ts

```diff
@@ -307,3 +308,14 @@ class ProgressStream extends stream.Transform {
     callback();
   }
 }
+
+interface Headers {
+  // tslint:disable-next-line no-any
+  [index: string]: any;
+}
+
+function populateAPIHeader(headers: Headers) {
+  const pkg = require('../../package.json');
+  const nodeVersion = process.version.replace(/^v/, '');
+  headers['x-goog-api-client'] = `gdcl/${pkg.version} gl-node/${nodeVersion}`;
```

Comment:
I assume this is for discovery clients (not gapic/grpc based)

Reply:
yeah, nodejs-googleapis-common is only used by our apiary clients:

https://www.npmjs.com/browse/depended/googleapis-common

(and a few forks of our apiary client).